src/ui/bookmarks_tab.rs

```diff
@@ -391,9 +392,21 @@ impl Component for BookmarksTab<'_> {
                 .title(" Bookmarks ")
                 .border_type(BorderType::Rounded);
             self.bookmarks_height = bookmarks_block.inner(chunks[0]).height;
+            let bookmark_count = lines.len();
             let bookmarks = List::new(lines).block(bookmarks_block).scroll_padding(3);
             *self.bookmarks_list_state.selected_mut() = current_bookmark_index;
             f.render_stateful_widget(bookmarks, chunks[0], &mut self.bookmarks_list_state);
+
+            // Scrollbar on left panel
```

Comment:
I think this is also unnecessary because we always show the details pane on the right

Reply:
I mean only comment, not sure if this is necessary